src/views/Login/Login.jsx

```diff
@@ -0,0 +1,10 @@
+import React from 'react'
+import LoginField from './login_field';
+
+const Login = () => {
+  return (
+    <LoginField/>
```

Comment:
Kindly use our "ReplicateOnInit" from Global Context, But I will allow this one for a while since you didn't had yet this function